sass/main.scss

```diff
@@ -225,3 +229,38 @@ img {
   }
 
 }
+
+.slide_members {
+  @extend .section;
+
+  h1 {
+    padding-bottom: 32px;
+  }
+
+  .member {
```

Comment:
Ill go remove some of the nesting where I can, just looked at the outputted css and I see what you mean